euxfel_h5tools/tests/test_reader.py

```diff
@@ -212,9 +220,9 @@ def test_stack_detector_data_2():
     print(comb.shape)
     assert comb.shape == (16, 60, 512, 128)
     for i in (0, 1, 2, 8, 9, 10):
-        assert (comb[i, ...] == data['SPB_DET_AGIPD1M-1/DET/{}CH0:xtdf'.format(i)]['image.data']).all()
+        assert np.testing.assert_equal(comb[i, ...], data['SPB_DET_AGIPD1M-1/DET/{}CH0:xtdf'.format(i)]['image.data'])
     for i in (3, 4, 5, 6, 7, 11, 12, 13, 14, 15):
-        assert (comb[i, ...] == np.zeros((60, 512, 128))).all()
+        assert np.testing.assert_equal(comb[i, ...], np.full((60, 512, 128), np.nan, dtype=np.float32))
```

Comment:
I think you probably don't need the `assert` before this, since the numpy function should throw an error if they're not equal.

It's probably better to rely on the numpy `assert_equal` method, rather than wrapping that to return True/False, because numpy can try to give some details in the error message if the assertion fails.